src/math/round.rs

```diff
@@ -35,3 +35,8 @@ pub fn round(mut x: f64) -> f64 {
         y
     }
 }
+
+#[test]
```

Comment:
Would you mind putting this behind a `#[cfg(test)]` ? Same below.

Reply:
Done.